app/Singletons/OsuAuthorize.php

```diff
@@ -2040,30 +2041,24 @@ public function ensureLoggedIn(?User $user, string $prefix = ''): void
      * @return string
      * @throws AuthorizationCheckException
      */
-    public function ensureCleanRecord(?User $user, string $prefix = ''): string
+    public function ensureCleanRecord(User $user, string $prefix = ''): void
     {
-        if ($user === null) {
-            return 'unauthorized';
-        }
-
         if ($user->isRestricted()) {
             throw new AuthorizationCheckException($prefix.'restricted');
         }
 
         if ($user->isSilenced()) {
             throw new AuthorizationCheckException($prefix.'silenced');
         }
-
-        return 'ok';
     }
 
     /**
      * @param User|null $user
```

Comment:
also needs updating, or just removed since it's not adding any new info, same with the other one